### ISSUE TITLE
SAP HANA XS Advanced, Connecting to the WebIDE and cloning a Git Repository to begin development

### ISSUE THREAD

**Issue by jglander** [SAPDocuments/Tutorials#1407]
Hi,

when cloning git repo <my-git-repo-name> (step 4, here: https://www.sap.com/developer/tutorials/xsa-connecting-webide.html) I get this error message in Web IDE console:

2:47:27 PM (git) Clone request failed Request failed: Gateway Timeout URI: /che/project/workspaceze3d9x60jrdool4v/import/<my-git-repo-name>?force=true

(I can clone the same git repo with my git bash but not in the Web IDE)

Any idea how to resolve this issue?

Regards, Jens

**Comment by Lsubatin** [comment]
Hi, Jens, this might help: https://archive.sap.com/discussions/thread/3945940

**Comment by jglander** [comment]
Hi Lsubatin,

Thanks for your hint. 

It looks as if this might help but unfortunately I'm not in the network (with my proxy) any more to verify it.
(But I wonder why I would have to do these additional steps, which Thomas Jung described, are necessary. I specified my proxies in the HANA 2.0, express edition installation, so I wonder that there are additional steps to configure which are not mentioned in the installation itself).

Anyhow I did complete new installation - being now in a network without any proxy - and there the described tutorial work fine so far :-)

Thanks again for your help.
Regards, Jens 

By the way: Is there any documentation how to switch a HANA express VM installation, which was initially installed with proxy environment, to an environment without proxy?
